react-frontend/src/components/Query/index.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import { useQuery } from '@apollo/react-hooks';
 
-const Query = ({ children, query, id }) => {
+const Query = ({ children, query, id, UID }) => {
   const { data, loading, error } = useQuery(query, {
-    variables: { id: id },
+    variables: { id: id, UID: UID },
```

Comment:
I may be nitpicking, lowercase 'id' and uppercase 'UID', we could have a uniform approach here.

Reply:
fixed